File: algoritmos/Algoritmo 2/Algoritmo2/src/main/java/com/fabio/algoritmo2/Elefante.java
package com.fabio.algoritmo2;

/**
 *
 * @author fabio
 */

public class Elefante extends Mamifero {
    
    public Elefante(double peso, int n_patas){
        super(peso, n_patas);
    }
    
    @Override
    public void comer(){
        System.out.println("estou comendo como um elefante.\n");
    }
}

File: algoritmos/Exercício 2/Exercicio2_UEPG/Classes/elefante.js
/*Elefante possui uma maneira de comer diferente dos demais.*/
class Elefante extends Mamifero{
    constructor(peso, patas) {
        super(peso, patas);
    }

    comer(){
        let mensagem = "Sou um elefante estou jogando comida dentro da boca";
        alert(mensagem);
        return mensagem
    }
}